seq2science/rules/quantification.smk

```diff
@@ -307,6 +307,12 @@ elif config["quantifier"] == "kallistobus":
                 -t {threads} -g {params.basename}_t2g.txt \
                 -o {output} -c1 {params.basename}_cdna_t2c.txt -c2 {params.basename}_intron_t2c.txt \
                 {params.barcode_arg} {params.options} {input.reads} > {log} 2>&1
+                # Validate output
+                if grep -q 'ERROR\|bad_alloc' "{log}"; then
+                  echo "An error was encoutered. Please check the logfile: {log} for details!" 
```

Comment:
`encountered`

Reply:
Also it's better not to print to shell because when running lots of stuff in parallel. Better to put some comment in the log, that we manually failed the rule (or no comment at all, I think the log is quite self-explanatory)